gap/PolygonalComplexes/modification.gi

```diff
@@ -2151,3 +2151,89 @@ end
 ##
 #######################################
 
+
+#######################################
+##
+##      Tori construction
+##
+
+
+InstallMethod( AllToriOfSimplicialSphere, 
+    "for a simplicial surface and an edge",
+    [IsSimplicialSurface],
+    function(surface)
+        local help_FaceMender,help_MendableEdgeAssignments,res,ee,ff,combFaces,edgesOfFaces,
+        orb,autGroup,faces,mendEdges,verticesOfFaces;
+        ## this method mends to faces of surface and removes the resulting face
+        ## from the constructed twisted polygonal complex to give rise to a simplcial surface 
+        help_FaceMender:=function(surface,faces,edges) 
```

Comment:
I think this method does quite the same as ConnectedFaceSum, maybe you can use the method if you want.

Reply:
ConnectedFaceSum has as input different surfaces and two flags. If both surfaces are spheres, the function also returns a sphere. Since I want to identify faces of the same sphere and construct a torus, I think ConnectedFaceSum can't be used in this situation.

Reply:
You are right!